### ISSUE TITLE
Importing on the renderer breaks on Electron forge.

### ISSUE THREAD

**Issue by adhambadr** [hardchor/electron-redux#102]
**Describe the bug**
It is mentioned once https://github.com/hardchor/electron-redux/issues/96, when using newer version of Electron or electron forge all ```import { ipcRenderer } from electron``` break whenever called from the renderer process. 

**To Reproduce**
Set up readme example with Electron forge & create-react-app instead of manual webpack 

**Expected behavior**
compile and run

**Screenshots**
see #96 

**Desktop (please complete the following information):**
 - OS: MacOS 10.14.3
 - Browser: Chrome
 - Version: 1.4.0

**Additional context**
tried replacing all impport {ipcRenderer } from 'electron' calls to :
```
const _electron = window ? window.require("electron") : require("electron") 
``` 
in all classes still somewhere the error persisted.

**Comment by hardchor** [comment]
@adhambadr I just published `1.5.1`. Could you check if that fixes your issue?

**Comment by hardchor** [comment]
Feel free to reopen if problem persists
